docs/design/managing-resources.md

```diff
@@ -48,7 +50,7 @@ spec:
         name: sleeper2
       spec:
         sleepFor: 4
-        wakeupMessage: "$(sleeper1/status/message)"
+        wakeupMessage: "$(sleeper1#status.message)"
```

Comment:
@ash2k does this format look fine to you? I wanted to decouple "resource name" reference from the JsonPath part.
`#` is like anchor on the GitHub documentation page section link: https://github.com/atlassian/smith#outputs